doc/changes/changes_4.0.5.md

```diff
@@ -1,6 +1,6 @@
-# Exasol Virtual Schemas 4.0.5, released 2020-??-??
+# Exasol Virtual Schemas 4.0.5, released 2020-12-22
 
-Code name: 
+Code name: Enabled `JOIN` capabilities in `MySQL` dialect adapter.
```

Comment:
In this release we don't have postgres dialect anymore, because it was moved. We should announce it everywhere. Here we ned to add a ## Summary part with a link to a new repository.
Please, also remove postgres driver from the  resources folder to tests, I think I saw it there recently.